rasa/shared/core/slots.py

```diff
@@ -265,8 +265,19 @@ def _as_feature(self) -> List[float]:
             # we couldn't convert the value to a list - using default value
             return [0.0]
 
+    @Slot.value.setter
+    def value(self, value: Any) -> None:
+        """Sets the slot's value."""
+        if not isinstance(value, list):
+            # Make sure we always store list items
+            value = [value]
+
+        Slot.value.fset(self, value)
```

Comment:
PyCharm compains of unexpected argument `value` in this line, I'm also not familiar with `fset` to understand why, but wouldn't it be easier just to assign `self._value = value`?
Also since this overrides the value.setter in Slot interface, shouldn't it also include `self._has_been_set = True`?

Reply:
Thanks for the hint! Fixed it and clarified the `fset` part. I'm calling the superclass property setter here to avoid duplicating `Slot`s `value.setter` logic. You can find more on it [here](https://docs.python.org/3/library/functions.html#property) or in [this Stackoverflow thread](https://stackoverflow.com/questions/44164816/overload-a-property-setter-in-python?noredirect=1&lq=1)

Reply:
Oh I see, so that's why there's no need to set `self._has_been_true` to `True`.